config/param.yml

```diff
@@ -5,7 +5,6 @@ motion_head: &id001
    labelref  : [0]
```

Comment:
Please leave this root file untouched -> you have the same file in PyQt/config 
Do you need to modify this one? If yes, rename it to param_GUI.yml